docs/content/contributing/guidelines-frontend.en-us.md

```diff
@@ -65,14 +65,17 @@ Recommended implementations:
 
 * Vue + Vanilla JS
 * Fomantic-UI (jQuery)
+* htmx
 * Vanilla JS
 
 Discouraged implementations:
 
 * Vue + Fomantic-UI (jQuery)
 * jQuery + Vanilla JS
+* htmx + anything else
```

Comment:
It doesn't work with "anything else". For example, if a page part uses JS "init" functions or Vue component, then it can't be reloaded by htmx, there is no clear way to call the "init" functions without side effect by htmx.

Reply:
What should I write instead? Maybe:
```md
* htmx + Vue
* htmx + jQuery
```

Reply:
Sorry I don't understand. How would htmx work with Vue or Fomantic-UI/jQuery ?

Reply:
I suggest to add the lines I wrote under `Discouraged implementations`

Reply:
Hmm .... I see, sorry I misunderstood, this section is "Discouraged implementations:"

Well, then `htmx + anything else` seems right, indeed it doesn't work with anything else at the moment ....

Sorry for my misunderstanding .....